src/exampleTypes/pivot.js

```diff
@@ -213,6 +213,9 @@ export default {
     let { type, node, value } = event
     if (type !== 'mutate') return
 
+    // Pause dispatches as a mutate (but also continue in case there are other properties being mutated)
+    if (F.matchesSignature(['paused'], value)) return
```

Comment:
this is so that a mutation with paused and something else like  `mutate({ paused: false, value: 'something' })` still works